client/src/components/course-view/AplusTokenDialog.tsx

```diff
@@ -0,0 +1,45 @@
+// SPDX-FileCopyrightText: 2024 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+// TODO: Currently unused
+
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  Link,
+  TextField,
+} from '@mui/material';
+
+type PropsType = {
+  handleClose: () => void;
+  open: boolean;
+};
+
+const AplusTokenDialog = ({handleClose, open}: PropsType): JSX.Element => {
+  return (
+    <Dialog open={open} onClose={handleClose}>
+      <DialogTitle>A+ API token</DialogTitle>
+      <DialogContent>
+        <p>
```

Comment:
All the same to me. I used a paragraph here because I noticed that in `FrontPage.tsx` we are also using a paragraph for displaying the "you have no courses" message.